android-kit-base/src/main/java/com/mparticle/kits/KitManagerImpl.java

```diff
@@ -1416,6 +1417,15 @@ public void execute(@NonNull String viewName,
         }
     }
 
+    public static String getValueIgnoreCase(Map<String, String> map, String searchKey) {
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            if (entry.getKey().equalsIgnoreCase(searchKey)) {
+                return entry.getValue();
```

Comment:
Should this return the `searchKey` instead?

Because if we are searching for `email`, and the entry value is actually `EMAIL`, then `EMAIL` would be returned, right?